infra/resources/src/main/res/values/strings.xml

```diff
@@ -424,4 +424,15 @@
     <!-- Notifications -->
     <string name="notification_sync_description">Sync in progress</string>
 
+    <!-- Identification pool validation -->
```

Comment:
Are these strings added to the translation service?

Reply:
They will be added automatically on merge.